src/Components/SinglePostCard/SinglePostCard.jsx

```diff
@@ -0,0 +1,43 @@
+import { useState } from 'react';
+import { BookMarkedIcon, BookmarkOutlineIcon, CommentsIcon, HeartFilledIcon, HeartOutlineIcon } from '../../Assets/AllSVG';
+import { AllPostComments } from './AllPostComments';
+
+const SinglePostCard = ({ photoUrl, displayName, post, timestamp, isBookMarked = false, isLiked = false }) => {
+	const [commentSection, setCommentSection] = useState(false);
+	const openComments = () => setCommentSection(true);
+	const closeComments = () => setCommentSection(false);
+
+	return (
+		<div className={`single-user-post ${commentSection && 'single-user-post-comments-opened'}`}>
+			{!isBookMarked ? <BookmarkOutlineIcon className='bookmark-icon' /> : <BookMarkedIcon className='bookmarked-icon' />}
```

Comment:
suggestion: better to keep `isBookMarked` and then put if else accordingly rather than keeping it `!isBookMarked`

Reply:
Sure will be changed in the next pr.